app/controllers/application_controller.rb

```diff
@@ -14,12 +14,17 @@ class ApplicationController < ActionController::Base
 
   attr_accessor :current_community
 
-  helper_method :current_community, :current_cluster, :multi_community?, :app_version
+  helper_method :current_community, :current_cluster, :multi_community?, :app_version, :nav_link_builder
+
+  def nav_link_builder
+    @nav_link_builder ||= Nav::LinkBuilder.new
+  end
```

Comment:
Looking good! This can be a protected method though, maybe even private.

Reply:
That works.